dev/ci/ci-coqhammer.sh

```diff
@@ -10,5 +10,7 @@ git_download coqhammer
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/coqhammer"
-  make
+  make tactics
+  make install-tactics
+  make plugin
```

Comment:
why this change?

Reply:
I did a couple of changes like this one. The problem was that just make would do more work on a second run (like re-run configure, or something like that)